index.html

```diff
@@ -4,10 +4,144 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="styles/styles.css">
     <title>Leaders' Week - 2023 - Home</title>
 </head>
 <body>
-    
+    <!-- MENU -->
+    <div class="nav-box d-flex">
+        <nav class="menu-top">
+            <a href="index.html"><img src="images/logo-gc-2.png" alt="Leaders Week Summit 2023 Logo" class="hidden-item logo"></a>
+            <ul>
+                <li><a href="#" class="menu-item icon"><img src="images/facebook-app-symbol.png" alt="facebook icon"></a></li>
+                <li><a href="#" class="menu-item icon"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                <li><a href="#" class="menu-item">English</a></li>
+                <li><a href="#" class="menu-item">My Page</a></li>
+                <li><a href="#" class="menu-item">Logout</a></li>
+            </ul>
+        </nav>
+        <img src="images/menu.png" alt="menu icon" id="menu-icon">
+        <nav class="menu-box" id="menu-box">
+            <a href="index.html"><img src="images/logo-gc-1.png" alt="Leaders Week Summit 2023 Logo" class="hidden-item logo"></a>
+            <ul>
+                <li><a href="about.html" class="menu-item active">About</a></li>
+                <li><a href="index.html#Program" class="menu-item">Program</a></li>
+                <li><a href="#" class="menu-item">Join</a></li>
+                <li><a href="#" class="menu-item">Sponsor</a></li>
+                <li><a href="#" class="menu-item">News</a></li>
+                <li><a href="index.html" class="menu-item hidden-item red-bordered">LW Company</a></li>
+            </ul>
+        </nav>
+    </div>
+    <!-- MAIN -->
+    <main>
+        <!-- HERO SECTION -->
+        <section class="hero-section d-flex" id="Home">
+            <div class="section-container d-flex">
+                <div class="title-box d-flex">
+                    <h1 class="title bold">Leaders Week Summit 2023</h1>
+                    <h2 class="subtitle bold"><q>Hello! Sharing world</q></h2>
+                </div>
+                <p class="description">
```

Comment:
- [ ] [Optional] Everything is good only a small change is required so I am giving it an optional one. Please increase the font weight of the heading tag to match the original design of both the home page and the About page.